### ISSUE TITLE
[BEIT v3] could not find "beit3_base_patch16_224.pth" file 

### ISSUE THREAD

**Issue by d007kkdl9999** [microsoft/unilm#1278]
**Describe**
Model I am using BEIT v3:

I am trying to finetune beit v3, but I could not find the available pretrained weights("beit3_base_patch16_224.pth" file ) from the document link.

(1) First, I download the pretrained weights from this link:
  https://github.com/microsoft/unilm/tree/master/beit3#download-checkpoints 
![image](https://github.com/microsoft/unilm/assets/46019238/839ae434-2291-458b-9d0b-6254f79d85bf)

(2) After that, I unzip this "zip" file which I just dowloaded, and I got the following files : 
![image](https://github.com/microsoft/unilm/assets/46019238/e0424051-5bd4-49eb-8209-cafea3d3d52a)

there is no "beit3_base_patch16_224.pth" file after unzipping this compressed  zip file. I am wondering where I am wrong , could you please help me ? Thanks !

**Comment by donglixp** [comment]
The downloaded file is '.pth' instead of '.zip'. You could use `wget -O beit3_base_patch16_224.pth http://original.link/to/the/file`.

**Comment by d007kkdl9999** [comment]
> The downloaded file is '.pth' instead of '.zip'. You could use `wget -O beit3_base_patch16_224.pth http://original.link/to/the/file`.

thanks for quick reply !

**Comment by lzzzzj** [comment]
> The downloaded file is '.pth' instead of '.zip'. You could use `wget -O beit3_base_patch16_224.pth http://original.link/to/the/file`.

Hello. I want to download <html>
<body>


TrOCR-Large-IAM | trocr-large-handwritten.pt
-- | --



</body>
</html>TrOCR-Large-IAM	[trocr-large-handwritten.pt](https://layoutlm.blob.core.windows.net/trocr/model_zoo/fairseq/trocr-large-handwritten.pt)

and I add " ?sv=2022-11-02&ss=b&srt=o&sp=r&se=2033-06-08T16:48:15Z&st=2023-06-08T08:48:15Z&spr=https&sig=a9VXrihTzbWyVfaIDlIT1Z0FoR1073VB0RLQUMuudD4%3D"
at the end of the link, but also download a zip file, too.
![image](https://github.com/microsoft/unilm/assets/57863550/a64e3436-1454-48b2-988e-39c1774eca15)



How to download the trocr-large-handwritten.pt file or change the trocr-large-handwritten.zip file into trocr-large-handwritten.pt form? Thanks!

**Comment by WalkerGoXQ** [comment]
> The downloaded file is '.pth' instead of '.zip'. You could use `wget -O beit3_base_patch16_224.pth http://original.link/to/the/file`.

thanks for the great work, but some questions about it: 
1. i tried to load .zip file directly, it works, and also i tried to change the file name from .zip to .pth and load it, it works too, i dont know which is better?     
2. if i want to use model beit3_base_itc_patch16_224.zip to generate image or text features with lower dimension such as 256-d out(768-d original), can i add TwoLayerMLP directly  as project layer, with no need to change others like loss?
